src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,5 +1,7 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
```

Comment:
this comment section is a javadoc of the `login()` method, which should be right above it, so let's do that, and the `userService` field along with the constructor field should be at the beginning of the class, in that exact order
[style guide](https://www.cs.cornell.edu/courses/JavaAndDS/JavaStyle.html#fielddecs)